albumentations/augmentations/transforms.py

```diff
@@ -242,6 +245,9 @@ def get_params(self):
         # Random int in range [0, 4)
         return {'factor': np.random.randint(0, 4)}
 
+    def apply_to_bbox(self, bbox, factor=0, **params):
+        return F.bbox_rot90(bbox, factor, **params)
+
```

Comment:
Could you please also update docstrings for `RandomRotate90` and `Transpose` and add `bboxes` to `Targets:` (after `image, mask`)?